vmdb/Gemfile

```diff
@@ -7,7 +7,7 @@ MiqBundler.include_gemfile("../lib/Gemfile", binding)
 # VMDB specific gems
 #
 
-gem "prototype-rails", "~>3.2.1"
+gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
```

Comment:
git based gems, at the moment, are causing no end of trouble for the build team. Please avoid them where possible.